index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+    <div class="container">
+    <head>
+        
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+    </head>    
+    <link rel="stylesheet" href="styles.css">
```

Comment:
I can see the head tag is inside the body section. Please move up before the body section including the style link and keep the header tag in the body there. In addition, please add a title that expresses your project page. The contents of a page title can have significant implications for search engine optimization.